src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -231,22 +229,22 @@ def get_storage_devices_topology(self):
         return topology
 
     async def get_gpus(self):
+
         if self.GPU:
             return self.GPU
 
         not_available = {'available': False, 'vendor': None}
-        cp = await run(['lshw', '-numeric', '-C', 'display'], check=False)
-        if cp.returncode:
-            self.logger.error('Unable to retrieve GPU details: %s', cp.stderr.decode())
+
+        try:
+            vga = subprocess.check_output('lspci | grep -i vga', shell=True).decode().strip().split('\n')
```

Comment:
Please use `run`, this will block because of being async.
Also can we please avoid using `shell` ?

Reply:
Fixed